workers/unity/Assets/Gdk/Core/Error/ForwardingDispatcher.cs

```diff
@@ -0,0 +1,67 @@
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class ForwardingDispatcher : ILogDispatcher
+    {
+        private Connection connection;
+
+#if UNITY_EDITOR
```

Comment:
There is no harm with logging this for headless workers. You can still see the logs in the unity log files. An additional benefit would be that a worker's behaviour is consistent between running in the editor and as standalone.